testonly/hammer/reader.go

```diff
@@ -120,7 +126,7 @@ func (o *validReadOps) getSMR(ctx context.Context, prng *rand.Rand) error {
 		return fmt.Errorf("failed to get-smr: %v", err)
 	}
 
-	err = o.smrs.pushSMR(*root)
+	err = o.sharedState.advertiseSMR(*root)
 	if err != nil {
```

Comment:
nit: Merge the two lines?

Reply:
Making the codebase shorter one line at a time.